test/box-luatest/gh_9979_tuple_format_validate_test.lua

```diff
@@ -111,3 +111,28 @@ g.test_tuple_format_validate = function()
         end
     end)
 end
+
+g.test_space_format_object = function()
+    g.server:exec(function()
+        local t = require('luatest')
+        local s = box.schema.space.create('test_fmt_obj', {
```

Comment:
Let's call the space `test`, and let's drop it in `after_test` hook (or `after_each`).